src/css/colors.scss

```diff
@@ -9,10 +9,12 @@ $defaultYellow: #F6C14D;
 $defaultGray: #858585;
 $grayTwo: #9B9B9B;
 $disabledGray: #B6B6B6;
+$grayThree: #CAC9CB;
 
 // Dark
 $black: #000000;
 $darkOne: #161616;
+$darkTwo: #161616f0;
```

Comment:
Is this hexadecimal correct?

Reply:
done

Reply:
I thought the hexadecimal has 6 characters, in this case, you put 8 characters, is ok?